### ISSUE TITLE
A little question about the compute_ca_loss

### ISSUE THREAD

**Issue by cats-food** [silent-chen/layout-guidance#12]
Hi, thanks for the nice work!

when computing the backward guidance loss, I noticed that in your code you only use the second half-batch of the attention maps by using 
https://github.com/silent-chen/layout-guidance/blob/3e169ac8f087f8623bed8c5c5968d9faadd8c7d8/utils.py#L12

```
def compute_ca_loss(attn_maps_mid, attn_maps_up, bboxes, object_positions):
    loss = 0
    object_number = len(bboxes)
    if object_number == 0:
        return torch.tensor(0).float().cuda() if torch.cuda.is_available() else torch.tensor(0).float()
    for attn_map_integrated in attn_maps_mid:
        attn_map = attn_map_integrated.chunk(2)[1]   # why chunk here?
...
```
I am a little bit confused, could you please give me some hint? thank you so much!

**Comment by silent-chen** [comment]
Hi, thanks for pointing this out! This was a mistake when I was organizing the code. I chunked it when it used to have the attention map of unconditional one. When I was updating the code, I found that I didn't have to compute the unconditional for guidance.  I will correct it now.
